### PR TITLE
build: Default to FusionAuth 1.32.1

### PR DESCRIPTION
**Pull request by ceefour**
**What this PR does / why we need it**:

So new user get current FusionAuth without specifying `image.tag` explicitly.

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR breaks earlier versions

### CODE REVIEW

**Comment by ceefour** [comment]
Due to recent bug with 1.32.0; I've changed the commit to (upcoming) 1.32.1

File: chart/examples/minikube/values.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ image:
   # image.repository -- The name of the docker repository for fusionauth-app
   repository: fusionauth/fusionauth-app
   # image.repository -- The docker tag to pull for fusionauth-app
-  tag: 1.30.1
+  tag: 1.32.1
```
Comment (drpebcak) [inline chart/examples/minikube/values.yaml:11]: This tag doesn't seem to exist yet in docker hub.

Comment (ceefour) [reply chart/examples/minikube/values.yaml:11]: It doesn't yet.. Waiting for @robotdan to publish it. :)

**Comment by robotdan** [comment]
@bguyza look ok to you?

**Review by drpebcak — APPROVED** [review]

**Comment by ceefour** [comment]
Yeay! https://hub.docker.com/layers/fusionauth/fusionauth-app/1.32.1/images/sha256-ee4e61195826720be7e9a64692c572a53785e585e5524362f8d43b448e2bded0?context=explore

**Comment by ceefour** [comment]
Thank you for the release @robotdan 🔥

**Comment by ceefour** [comment]
(and very grateful FusionAuth isn't affected by Log4shell) 🙏

**Comment by robotdan** [comment]
I can merge this PR. 

Is this something we should just be versioning with the release of FusionAuth? If so, we could just add this to our release procedures. Thoughts?

**Review by robotdan — APPROVED** [review]
LGTM

**Comment by ceefour** [comment]
Helm is increasingly become the preferred deployment mechanism in Kubernetes, and Kubernetes is increasingly become the preferred architecture for all software in organizations.

So I I'd agree in Helm tracking latest stable FusionAuth.

Also, you can give commercialization notes in Helm chart's notes (e.g. For enterprise sales & support, please refer to www....)

**Pull request merged**